datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxEventEndpoint.java

```diff
@@ -40,4 +42,13 @@
             @QueryParam("types") List<StructuredEventType> types,
             @QueryParam("page") @DefaultValue("0") Integer page,
             @QueryParam("size") @DefaultValue("100") Integer size);
+
+    @GET
+    @Path("zip")
+    @Produces(MediaType.APPLICATION_OCTET_STREAM)
+    @ApiOperation(value = LIST_FOR_RESOURCE_ZIP, produces = MediaType.APPLICATION_JSON, notes = AUDIT_EVENTS_NOTES,
```

Comment:
MediaType.APPLICATION_JSON ->  it should be MediaType.APPLICATION_OCTET_STREAM

Reply:
addressed